Trappist/src/Promact.Trappist.DomainModel/Models/Test/TestIpAddress.cs

```diff
@@ -0,0 +1,16 @@
+﻿
+using System.ComponentModel.DataAnnotations.Schema;
+
+
+namespace Promact.Trappist.DomainModel.Models.Test
+{
+    public class TestIpAddress : BaseModel
+    {
+        public string IPAddress { get; set; }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename property 'IPAddress' to match camel case naming rules, consider using 'IpAddress'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS100)